tests/ovirt/requests/compute/create_vm_tests.rb

```diff
@@ -1,26 +1,14 @@
 Shindo.tests("Fog::Compute[:ovirt] | vm_create request", 'ovirt') do
-
   compute = Fog::Compute[:ovirt]
   name_base = Time.now.to_i
 
   tests("Create VM") do
     response = compute.create_vm(:name => 'fog-'+name_base.to_s, :cluster_name => 'Default')
-    test("should be a kind of OVIRT::VM") { response.kind_of?  OVIRT::VM}
+    test("should be a kind of OVIRT::VM") { response.is_a? OVIRT::VM}
   end
 
   tests("Create VM from template (clone)") do
     response = compute.create_vm(:name => 'fog-'+(name_base+ 1).to_s, :template_name => 'hwp_small', :cluster_name => 'Default')
-    test("should be a kind of OVIRT::VM") { response.kind_of?  OVIRT::VM}
-  end
-
-  tests("Fail Creating VM") do
-    begin
```

Comment:
I can't wrap my head around how this was supposed to work. I'm all in for fixing it rather than deleting it, if we can figure out a way to make it test something meaningful

Reply:
It might be a name that used to exist in the mock files we use. Maybe calling `create_vm` twice with the same parameters would actually get that error.

Reply:
Calling it several times in a row just returns the same response